Lombiq.Hosting.Tenants.FeaturesGuard/Extensions/OrchardCoreBuilderExtensions.cs

```diff
@@ -21,6 +21,23 @@ public static OrchardCoreBuilder ConfigureFeaturesGuardForAzureStorage(this Orch
         return builder;
     }
 
+    /// <summary>
+    /// Binds Elasticsearch as the conditional feature and Search or Indexing as the condition feature to
+    /// <see cref="ConditionallyEnabledFeaturesOptions"/>.
+    /// </summary>
+    public static OrchardCoreBuilder ConfigureFeaturesGuardForElasticsearch(this OrchardCoreBuilder builder)
+    {
+        builder.ConfigureServices((tenantServices, _) =>
+            tenantServices.PostConfigure<ConditionallyEnabledFeaturesOptions>(options =>
+                options.EnableFeatureIfOtherFeatureIsEnabled = new Dictionary<string, IEnumerable<string>>
```

Comment:
Change the logic so if the dictionary EnableFeatureIfOtherFeatureIsEnabled already exists add the `["OrchardCore.Search.Elasticsearch"] =
                        new List<string> { "OrchardCore.Search", "OrchardCore.Indexing" },` key value pair into it, instead of overriding the value to this new dictionary.

Same for `ConfigureFeaturesGuardForAzureStorage()`. Then you can use both functions after each other.

Reply:
Good idea!

Reply:
Also I think you can dry these two together.

Reply:
> Also I think you can dry these two together.

I don't want to put them into one configure function, because we have a separate `if` for adding the Azure configuration: 
![image](https://user-images.githubusercontent.com/80963259/207899342-f4437764-a60f-4903-81a2-5431478be7e1.png)

So not always both are enabled.

Reply:
Yeah I know. But you can DRY it without using both.